static/index.html

```diff
@@ -117,6 +118,39 @@ <h3>Current support</h3>
 							<li>VYV Photon <span class="badge badge-success">New</span>
 						</ul>
 
+						<h3>Planned support</h3>
+						<ul>
+							<li>Arkaos Grand VJ / Mediamaster</li>
+							<li>Avolites AI</li>
+							<li>Barco Imagepro</li>
+							<li>Barco Projectors</li>
+							<li>Chamsys MagicQ</li>
+							<li>Dante Audio Routing</li>
+							<li>Ember+</li>
+							<li>Entec DMXPro</li>
+							<li>Extron Matrix Routers</li>
+							<li>Extron Presentation Switchers</li>
+							<li>GrassValley Switchers</li>
+							<li>Green Hippo</li>
+							<li>GreenGo Digital</li>
+							<li>Kramer Matrix Routers</li>
+							<li>Kramer Presentation Switchers</li>
+							<li>NEC Projectors</li>
+							<li>Nevion Routers</li>
```

Comment:
Wanna move this to current support?

Reply:
The only stuff that should be updated in current support are the Global Cache and Oliver in contributor. Some of the other things in may have been changed/fixed since your last update with the latest release. This pull request has been there for a while. I may not have had a good merge with your new. I can go back and overwrite my file with the current and then add and readjust just what I wanted to. I may have time later to fix that. I don't want to screw up the current lists.